tools/config/go-templates/model_simple.mustache

```diff
@@ -1,454 +0,0 @@
-// checks if the {{classname}} type satisfies the MappedNullable interface at compile time
```

Comment:
The actual impact of this change. We no longer have issues with missing properties as oneOfs are everywhere and we validating them. I will restore actual template file in separate PR (we need file from the repo)